modules/nw-creating-a-route.adoc

```diff
@@ -0,0 +1,73 @@
+[id="creating-a-route"]
+= Creating a route
+
+
+
+A route allows you to host your application at a public URL. It can either be secure or insecure, depending on the network security configuration of your application. An insecure route uses the basic HTTP routing protocol and exposes a service on an insecure application port.
+You can optionally define security, such as TLS, for a route. To know how to create a secure route, see https://docs.openshift.com/container-platform/4.4/networking/routes/secured-routes.html[secure routes].
```

Comment:
Saying "security, such as TLS" suggests there are alternatives to using TLS, but TLS is the crux of security for routes.  Also, shouldn't we use relative links?  
```suggestion
You can optionally configure a route to use TLS for security. To know how to create a secure route, see xref:../../networking/routes/secured-routes.html[secure routes].
```

Reply:
@Miciah I have rephrased it.